public/app/features/dashboard-scene/scene/layout-rows/RowItemsEditor.tsx

```diff
@@ -23,10 +23,10 @@ function RowHeaderCheckboxMulti({ model }: { model: RowItems }) {
   let indeterminate = false;
 
   for (let i = 0; i < rows.length; i++) {
-    const { isHeaderHidden } = rows[i].useState();
+    const { hideHeader: isHeaderHidden } = rows[i].useState();
```

Comment:
would rename the local variable here as well

Reply:
Ah, good catch.